### PR TITLE
Insert icons from breeze style inside ephyveiwer.

### PR DESCRIPTION
**Pull request by samuelgarcia**
@jpgill86 

I made icons system for ephyviwer.

The idea is: when you want to insert an icons you:
 * put the media files inside /icons/img directory
 * then launch "make_ressource.py" this modify the "icons_py3.py" (binary)
 * then you can access icons as a QT ressource with ":/icons.png"

### CODE REVIEW

File: ephyviewer/icons/make_ressources.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+
+from tridesclous.gui.myqt import QT_MODE
```
Comment (jpgill86) [inline ephyviewer/icons/make_ressources.py]: This can be changed to `from ephyviewer.myqt import QT_MODE`, right?

**Comment by jpgill86** [comment]
Do we also need
```python
packages = ['ephyviewer', 'ephyviewer.datasource', 'ephyviewer.tests', 'ephyviewer.icons'],
```
in setup.py?

**Comment by samuelgarcia** [comment]
Oups thank you for your proff reading.

**Comment by jpgill86** [comment]
I assume you don't have any more changes for this PR, so I'm going to merge.

**Pull request merged**